training/gordon-group/src/entity/SimulationModelOutput.c3typ

```diff
@@ -0,0 +1,21 @@
+/**
+* SimulationModelOutput.c3typ
+* A single output taken from a single [SimulationSample](type:SimulationSample)
+*/
+@db(datastore='Cassandra',
+    partitionKeyField='parent',
```

Comment:
I think we might want to add the `latitude` and `longitude` fields to the partition key like:
```
patitionKeyField='parent,latitude,longitude'
```
So that they can be filtered on